bioblend/galaxy/workflows/__init__.py

```diff
@@ -88,39 +88,56 @@ def get_workflow_inputs(self, workflow_id, label):
         inputs = wf['inputs']
         return [id for id in inputs if inputs[id]['label'] == label]
 
-    def import_workflow_dict(self, workflow_dict):
+    def import_workflow_dict(self, workflow_dict, publish=False):
         """
         Imports a new workflow given a dictionary representing a previously
         exported workflow.
 
         :type workflow_dict: dict
         :param workflow_dict: dictionary representing the workflow to be imported
+
+        :type publish: bool
+        :param publish:  if ``True`` the uploaded workflow will be sharable;
+                         otherwise it will be visible only by the user which uploads it (default)
         """
-        payload = {'workflow': workflow_dict}
+        payload = {'workflow': workflow_dict, 'publish': publish}
 
         url = self.gi._make_url(self)
         url = _join(url, "upload")
         return self._post(url=url, payload=payload)
 
-    def import_workflow_json(self, workflow_json):
+    def import_workflow_json(self, workflow_json, publish=False):
         """
         .. deprecated:: 0.9.0
            Use :meth:`import_workflow_dict` instead.
+
+        :type workflow_json: dict
+        :param workflow_json: dictionary representing the workflow to be imported
+
+        :type publish: bool
+        :param publish:  if ``True`` the uploaded workflow will be sharable;
+                         otherwise it will be visible only by the user which uploads it (default)
+
         """
-        return self.import_workflow_dict(workflow_json)
+        return self.import_workflow_dict(workflow_json, publish)
```

Comment:
Can you revert all the changes to this deprecated method, please?

Reply:
All the changes reverted via https://github.com/galaxyproject/bioblend/pull/247/commits/d42de5d02f211ca90fad00a8bf42173005aa9455.
To support the new feature in the `import_workflow_from_local_path` method, I've removed the dependency to the deprecated `import_workflow_json` (see https://github.com/galaxyproject/bioblend/pull/247/commits/7e5b873c292e3afe01c0ceaaddbf74ba938524f8).

Reply:
Perfect, thanks!